### PR TITLE
[incubator/external-service-operator] Introduction

### PR DESCRIPTION
**Pull request by alwinmarkcf**
Signed-off-by: Alwin Mark <a.mark@crowdfox.com>

#### What this PR does / why we need it:

This is a new Chart for deploying the external-service-operator. The external-service-operator is able to publish services into a namespace which are actually running outside of the Kubernetes Cluster, while performing healthchecks and creating ingress resources as well.

#### Which issue this PR fixes
 No issue yet.

#### Special notes for your reviewer:

This is my first chart I contribute of a service we are running a Crowdfox GmbH on production. I already contributed to other Charts like Traefik, but I don't know how the Maintainership of new Charts will be handled. Basically this should not be a big deal as the whole logic is inside of the container or the CRD so there is not much to configure.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by alwinmarkcf** [comment]
@scottrigby @jlegrone @maorfr could you please review this?

**Comment by maorfr** [comment]
hey @alwinmarkcf 

Have you considered hosting this chart and adding it to helm hub?
https://github.com/helm/hub/blob/master/Repositories.md

**Comment by alwinmarkcf** [comment]
Ok I'll give it a shot. Created an own repository and added it to the hub (PR pending):
https://github.com/helm/hub/pull/128

File: incubator/external-service-operator/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+appVersion: "1.0"
+description: |
+  A Helm chart for deploying External-Service-Operator which is repsonsible for creating and monitoring Services running outside of the cluster.
+  The External-Service-Operator will create K8s Ressources for those Services, so they can be used, as they would just live inside the cluster.
+  One of the Usecases can also be, that you want to expose an external Service via your Kubernetes Ingress Controller in order to have one central reverse Proxy.
+name: external-service-operator
+version: 0.0.1
+home: https://github.com/CrowdfoxGmbH/external-service-operator
+keyworkds:
```
Comment (deepanshululla) [inline incubator/external-service-operator/Chart.yaml]: keywords*

**Pull request closed without merging**